### DISCUSSION TITLE
wordwrap, number of columns

### DISCUSSION THREAD

**Discussion by pjfichet** [scandum/tintin#132] [Script help]
Hello alls,

Is there a way to define a maximum number of column, or force word wrapping at 80 characters ?

If `#split 0 0 0 -80` is the command to use, is it possible to remove the vertical line it draws ?

Thank you !

**Comment by scandum** [comment]
There's no explicit way to change the wrapping width.

To not draw the vertical line try:
```
#line quiet #split 0 0 0 -80
```

**Reply by pjfichet** [reply]
Perfect, thank you !

**Comment by Ulenspiegel** [comment]
Seems like prepending #line quiet prevents all lines from being drawn, not just the vertical one. I wonder, what happened with #config wordwrap with number argument , it was introduced at some point in 2.01.91, but then was removed?

**Comment by scandum** [comment]
I probably removed it as it became obsolete with vertical split support.
